### ISSUE TITLE
Install for XM Only

### ISSUE THREAD

**Issue by GreyGhostStudio** [bartverdonck/Sitecore-Forms-Extensions#94]
Would be great to have a version of this for install that doesn't require XP to be installed. Of course with a smaller footprint of features that don't apply to this specific version of Sitecore.

I know I know.. "clients buy Sitecore as XM only?!?".

**Comment by markgibbons25** [comment]
Should be as simple as adding some Tracker.Enabled and xDB.Enabled checks

**Comment by bartverdonck** [comment]
Not really, an XM version would contain less functionality so a number of Sitecore items shouldn't be included as well. So in effect, you would have 2 deliverables, one for xp and one for xm.
